doc/reference.md

```diff
@@ -100,6 +100,7 @@ Parameter       | Type                                     | Description
 :-------------- | :--------------------------------------- | :-----------
 onCardNonceRequestSuccess | [CardEntryNonceRequestSuccessCallback](#cardentrynoncerequestsuccesscallback) | Invoked when card entry is completed and the SDK has processed the payment card information.
 onCardEntryCancel | [CardEntryCancelCallback](#cardentrycancelcallback) | Invoked when card entry is canceled.
+collectPostalCode | bool                                   | Indicates that the customer must enter the postal code associated with their payment card. When false, the postal code field will not be displayed. Defaults to `true`.<br>**Notes**: Postal code collection is required for processing payments for Square accounts based in the United States, Canada, and United Kingdom. Disabling postal code collection in those regions will result in all credit card transactions being declined.
```

Comment:
"collection" is normally a noun and for a developer, it can mean a list, hash, map...
Maybe it is better to say "the postal code input field"  instead of "postal code collection"